packages/web_stack/setup.cfg

```diff
@@ -32,6 +32,7 @@ version = 23.1.dev0
 [options]
 include_package_data = True
 install_requires =
+    galaxy-data
```

Comment:
I don't like this - but handlers.py has a ``galaxy.model.base import transaction`` import so it does accurately reflect what is going on. Thank you for the fix.

Reply:
And `from galaxy.model import database_utils` in `lib/galaxy/web_stack/__init__.py`